cli/cmd/errors.go

```diff
@@ -211,3 +212,10 @@ func ErrorDuplicateCLIEnvNames(environment string) error {
 		Message: fmt.Sprintf("duplicate environment names: %s is defined more than once", s.UserStr(environment)),
 	})
 }
+
+func ErrorInvalidOperatorEndpoint(endpoint string) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrInvalidOperatorEndpoint,
+		Message: fmt.Sprintf("%s is not a cortex operator endpoint, run `cortex cluster info` to show your operator endpoint or spin up a new cluster with `cortex cluster up`", endpoint),
```

Comment:
nit (if you agree): 

"%s is not a cortex operator endpoint; run `cortex cluster info` to show your operator endpoint or run `cortex cluster up` to spin up a new cluster"